src/node_contextify.cc

```diff
@@ -161,6 +162,9 @@ class ContextifyContext {
         clone_property_method->Call(global, arraysize(args), args);
       }
     }
+
+    if (try_catch.HasCaught())
+      try_catch.ReThrow();
```

Comment:
You don't need to ReThrow() if you drop the TryCatch, the exception will simply bubble up.

(I believe the caller of CopyProperties() is doing that wrong, too.)